src/script/browser/CheckBrowser.ts

```diff
@@ -17,119 +17,86 @@
  *
  */
 
-import {QUERY_KEY} from '../auth/route';
+import Cookies from 'js-cookie';
 
-const isMobile = (): boolean => 'ontouchstart' in window || !!navigator.maxTouchPoints;
+import {QUERY_KEY} from '../auth/route';
 
 const isOauth = (): boolean => location?.hash?.includes(QUERY_KEY.SCOPE) ?? false;
 
-const supportsCookies = (): boolean => navigator.cookieEnabled;
-/* return new Promise((resolve, reject) => {
-     switch (navigator.cookieEnabled) {
-       case true:
-         return resolve();
-       case false:
-         return reject(new Error());
-       default:
-         Cookies.set(cookieName, 'yes');
-         if (Cookies.get(cookieName)) {
-           Cookies.remove(cookieName);
-           return resolve();
-         }
-         return reject(new Error());
-     }
-   });*/
+const cookieName = 'cookie_supported_test_wire_cookiename';
 
-const supportsIndexDB = (): boolean => 'indexedDB' in window;
-/*{
-    let supportIndexedDb;
-    try {
-      supportIndexedDb = !!window.indexedDB;
-    } catch (error) {
-      supportIndexedDb = false;
-    }
-    if (!supportIndexedDb) {
-      return Promise.reject(new Error('IndexedDB not supported'));
+const supportsCookies = (): Promise<boolean> =>
+  new Promise<boolean>((resolve, _reject) => {
+    switch (navigator.cookieEnabled) {
+      case true:
+        resolve(true);
+        break;
+      case false:
+        resolve(false);
+        break;
+      default:
+        Cookies.set(cookieName, 'yes');
+        if (Cookies.get(cookieName)) {
+          Cookies.remove(cookieName);
+          resolve(true);
+        } else {
+          resolve(false);
+        }
     }
+  });
```

Comment:
why is this one async, I cannot see any async operation in there 🤔

Reply:
right! I just copied it from the app and didn't think to double-check!
done